docs/source/load-balancing/dc-robin.md

```diff
@@ -25,9 +29,9 @@ let local_dc_name: String = "us_east".to_string();
 
 let session: Session = SessionBuilder::new()
     .known_node("127.0.0.1:9042")
-    .load_balancing(Arc::new(DcAwareRoundRobinPolicy::new(local_dc_name)))
+    .load_balancing(Arc::new(DcAwareRoundRobinPolicy::new(local_dc_name, true)))
     .build()
     .await?;
 # Ok(())
 # }
-```
\ No newline at end of file
+```
```

Comment:
Do you know why GitHub marks this line as a change?

Reply:
It's a matter of whitespace: my editor automatically added a newline, which had been missing at the end of file.